From da1e9a3807d2b04b7174f31903939b12397bb85c Mon Sep 17 00:00:00 2001
From: Daniel Fullmer <danielrf12@gmail.com>
Date: Thu, 7 May 2020 17:52:03 -0400
Subject: [PATCH 1/3] Remove LineageOS keys

Signing target files would fail:
extra recovery-only key(s): vendor/lineage/build/target/product/security/lineage.x509.pem
However, that key is not available in our build environment. We don't
want to include them anyway.
---
 config/common.mk | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index 2fe9cb01..3de8f1c0 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -197,12 +197,7 @@ PRODUCT_PACKAGE_OVERLAYS += \
 PRODUCT_PACKAGES += \
     TrebuchetOverlay
 
-PRODUCT_EXTRA_RECOVERY_KEYS += \
-    vendor/lineage/build/target/product/security/lineage
-
 include vendor/lineage/config/version.mk
 
--include vendor/lineage-priv/keys/keys.mk
-
 -include $(WORKSPACE)/build_env/image-auto-bits.mk
 -include vendor/lineage/config/partner_gms.mk
-- 
2.27.0

